osu.Game/Screens/Backgrounds/BackgroundScreenDefault.cs

```diff
@@ -102,6 +106,18 @@ private Background createBackground()
             else
                 newBackground = new Background(backgroundName);
 
+            switch (showSeasonalBackgrounds.Value)
+            {
+                case SeasonalBackgrounds.Sometimes:
+                    if (seasonalBackgroundLoader.IsInSeason())
+                        goto case SeasonalBackgrounds.Always;
+                    break;
+
+                case SeasonalBackgrounds.Always:
+                    newBackground = seasonalBackgroundLoader.LoadBackground() ?? newBackground;
+                    break;
+            }
```

Comment:
this should probably be above everything else in this method. as it is, a different background is constructed above, then potentially thrown away due to the current sequence.